bedrock/firefox/templates/firefox/nightly_whatsnew.html

```diff
@@ -74,12 +77,6 @@ <h2>{{ self.page_title() }}</h2>
               karma if you add the <em>nightly-community</em> keyword to your bug reports)!
             {% endtrans %}
           </p>
-          <p>
-            {% trans irc='ircs://irc.mozilla.org/nightly' %}
-              You can also interact with the Nightly community on IRC in the <a href="%(irc)s">#nightly</a> channel.
-              Feel free to ask questions and talk about features, regressions or crashes you may be experiencing.
-            {% endtrans %}
-          </p>
```

Comment:
I wonder why `chat.mozilla.org` wasn't picked up here?

Reply:
Because it didn't exist at the time and we had not moved to Matrix.

Reply:
Should I open up a new bug for it?

Reply:
Yes please (and CC me), thanks!

Reply:
Did that: https://github.com/mozilla/bedrock/issues/8688

Sorry for the long wait...